src/tendencies/implicit/implicit_tendency.jl

```diff
@@ -9,28 +9,24 @@ import ClimaCore: Fields
 # TODO: Can we rewrite ᶠfct_boris_book and ᶠfct_zalesak so that their broadcast
 # expressions are less convoluted?
 vertical_transport!(ᶜρχₜ, ᶜJ, ᶜρ, ᶠu³, ᶜχ, dt, ::Val{:none}) =
-    @. ᶜρχₜ = -(ᶜdivᵥ(ᶠwinterp(ᶜJ, ᶜρ) * ᶠu³ * ᶠinterp(ᶜχ)))
+    @. ᶜρχₜ = -(ᶜadvdivᵥ(ᶠwinterp(ᶜJ, ᶜρ) * ᶠu³ * ᶠinterp(ᶜχ)))
```

Comment:
I think this one can be left as `ᶜdivᵥ`. If this does get changed, it might be good to also change the corresponding Jacobian term on line 32 of `wfact.jl`.

Reply:
Okay. I wanted to keep them all consistent.